README.md

```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
 ### Using møte
 
 Dependencies:
- - Optional: `memcached` (`sudo dnf install memcached`)
- - Python 2.7.x
+ - `memcached` (`sudo dnf install memcached`)
+ - Python 3.8.X
+ - Pipenv
 
 Installation:
- - Optional: create virtual environment
- - `pip install -r requirements.txt`
- - Install using `python setup.py develop`
+ - `pipenv install` will create a new virtual environment and install the requirements. `
 
 Running møte:
+ - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+   - You can do it by running the following command in command line too: `sed -i '/use_memcached = / s/False/True/' mote/config.py`
+ - Enter the Pipenv's newly created virtualenv's shell with `pipenv shell`
  - Run `python runmote.py`
 
+**Note :** Make sure `memcached` is running in the background.
```

Comment:
It is out of scope for this Pull Request, but maybe we can use `docker-compose` or `podman-compose` to make this easier to set up and run for new developers.

Reply:
I was thinking to something similar, I don't think if we would need `docker-compose` / `podman-compose` , we can proceed with a simple `Dockerfile` too. What do you think ?

Reply:
Here's an issue to keep track of it: https://github.com/fedora-infra/mote/issues/142